From 0be7b88d66a64d6d734ea025f9f8aeacbbe588d0 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Tue, 13 Feb 2018 09:59:38 +0800
Subject: [PATCH] drivers/input/touchscreen/cy8c_touchpad: fix compile warning

drivers/input/touchscreen/cy8c_touchpad.c:77:20: warning: xfer_msg[1].flags
is used uninitialized in this function [-Wuninitialized]

Change-Id: I2d1f199f098b38af92037e2ebbd63c05fc4a1f35
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/input/touchscreen/cy8c_touchpad.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/touchscreen/cy8c_touchpad.c b/drivers/input/touchscreen/cy8c_touchpad.c
index 87a1962ae566..8088cb918c79 100644
--- a/drivers/input/touchscreen/cy8c_touchpad.c
+++ b/drivers/input/touchscreen/cy8c_touchpad.c
@@ -74,7 +74,7 @@ static u32 i2c_cy8ctouch_read(struct cy8c_touch *ts,  u8 *buf, u32 num)
 
 	xfer_msg[1].addr = client->addr;
 	xfer_msg[1].len = num;
-	xfer_msg[1].flags |= I2C_M_RD;
+	xfer_msg[1].flags = I2C_M_RD;
 	xfer_msg[1].buf = buf;
 
 	ret = i2c_transfer(client->adapter, xfer_msg, ARRAY_SIZE(xfer_msg));
-- 
2.35.3

